generalhw/adding-vnc-setup.md

```diff
@@ -189,80 +189,37 @@ systemctl enable --now kvmd-vnc
 
 ### Switch PiKVM to RGB24 video capture format
 
-If needles don't match due to incorrect colors, that's probably because `UYVY`
-format is used (see output of `v4l2-ctl --list-formats`).
+If needles don't match due to incorrect colors (most likely black isn't black
+but very dark shade of gray), that's probably because `UYVY` format is used (see
+output of `v4l2-ctl --list-formats`).
 
 In this case, copy `kvmd.streamer.cmd` part of `/etc/kvmd/main.yaml` to
-`/etc/kvmd/override.yaml` and change `--format=` parameter to `RGB24` and
-`--encoder=` to `cpu`.  This part will look like this:
+`/etc/kvmd/override.yaml` and change `--format=` parameter to `rgb24` and
+`--encoder=` to `cpu`.  Due to [bug in TC358743](https://github.com/raspberrypi/linux/issues/6068),
+red and blue channels will be swapped.  It isn't enough to use `bgr24` - it
+would be properly communicated to the controller, so it will swap the channels
+as well as metadata about used format.  To handle this problem, a new option
+`--format-swap-rgb` has to be added.
```

Comment:
```suggestion
`--format-swap-rgb` has to be added (available since ustreamer v6.8).
```
Worth mentioning in case an older ustreamer is installed given that the option is 10 months old.

Reply:
https://github.com/3mdeb/openqa-tests-qubesos/compare/c91f3f4dccd0ec59c0a56ad523bb3f9430a1a6df..fee8141d25c417c5b87babaa21c665c0f035762f